src/Global/molecules/UserMenu.tsx

```diff
@@ -220,6 +219,7 @@ const UserMenu: React.FC<IUserMenu> = ({
               <UserDetails
                 onUserDrawerMetaClick={onUserDrawerMetaClick}
                 key={key}
+                dataObjKey={key}
```

Comment:
Thanks for the changes, just one thing please also change this name to something like `userMetaIndex` .